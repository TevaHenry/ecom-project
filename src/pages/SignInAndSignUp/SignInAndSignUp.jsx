import React from 'react';
import SignIn from '../../components/SignIn/SignIn';
import './SignInAndSignUp.scss';

const SignInAndSignUp = (props) => (
  <div className='sign-in-and-sign-up'>
    <SignIn />
  </div>
)

export default SignInAndSignUp;