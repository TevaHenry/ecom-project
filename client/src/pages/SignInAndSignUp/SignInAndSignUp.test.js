import { shallow } from 'enzyme';
import React from 'react';
import SignInAndSignUp from './SignInAndSignUp';

it('expect to render SignInAndSignUp component', () => {
  expect(shallow(<SignInAndSignUp />)).toMatchSnapshot()
})