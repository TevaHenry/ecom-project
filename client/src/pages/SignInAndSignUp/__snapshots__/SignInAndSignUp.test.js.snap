// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SignInAndSignUp component 1`] = `ShallowWrapper {}`;
